docs/layers/terrain-layer.md

```diff
@@ -140,25 +136,25 @@ The default value of `elevationDecoder` decodes a grayscale image:
 
 ##### `bounds` (Array, optional)
 
-Bounds of the image to fit x,y coordinates into. In [minX, minY, maxX, maxY] world coordinates. Must be supplied when using as a Non-Tiled terrain, otherwise optional.
+Bounds of the image to fit x,y coordinates into. In `[left, top, right, bottom]` world coordinates. Must be supplied when using non-tiled elevation data.
```

Comment:
The left, top, right, bottom notation is confusing to me. For example, I don’t remember what top means - I think indicating it’s the “min y” is helpful. This is especially confusing when different tools order lat/lng differently.

Reply:
minY/maxY refer to different things in different context. In a geospatial viewport, the minLat is actually at the bottom.

Reply:
That makes sense. Would you be able to clarify the relation between min/max lat/lng and top/bottom/left/right in the docs?

Reply:
Next line in doc